azure/ConsiderationsForServiceDesign.md

```diff
@@ -116,8 +116,6 @@ For example, `collectedItems` not `itemsCollected`
 
 For example, `nextUrl` not `nextURL`.
 
-:ballot_box_with_check: **YOU SHOULD** use "DateTime" suffix in names of `date-time` values.
```

Comment:
What should they use then? We see such a mix of different names, it's a bit of a quagmire from library to library of "Date", "DateTime", "Timestamp", "TimeStamp" (that's just wrong), and nothing. Surely, we could strive for some consistency.

Reply:
Opened #409 to track.